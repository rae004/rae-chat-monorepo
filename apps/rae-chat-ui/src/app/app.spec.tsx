import { render } from '@testing-library/react';

import App from './app';

describe('Rae Chat UI', () => {
    it('should render successfully', () => {
        const { baseElement } = render(<App />);
        expect(baseElement).toBeTruthy();
    });

    it('should match last snapshot', () => {
        const { container } = render(<App />);
        expect(container).toMatchSnapshot();
    });
});
