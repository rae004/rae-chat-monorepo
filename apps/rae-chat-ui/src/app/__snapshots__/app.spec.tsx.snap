// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Rae Chat UI > should match last snapshot 1`] = `
<div>
  <div
    class="_title_1346f5"
  >
    Hello!
     
    <span
      aria-label="Waving hand emoji"
      role="img"
    >
      ðŸ‘‹
    </span>
  </div>
</div>
`;
